### PR TITLE
add stuff

### PR DESCRIPTION
**Pull request by mullr**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/mullr/code-review-test/pull/1%23issuecomment-111579380%22%2C%20%22https%3A//github.com/mullr/code-review-test/pull/1%23issuecomment-111579426%22%2C%20%22https%3A//github.com/mullr/code-review-test/pull/1%23discussion_r32339849%22%2C%20%22https%3A//github.com/mullr/code-review-test/commit/e2ba0277862212ae04b53994c5120f14cf57fc36%23commitcomment-11659416%22%2C%20%22https%3A//github.com/mullr/code-review-test/commit/e2ba0277862212ae04b53994c5120f14cf57fc36%23commitcomment-11659461%22%2C%20%22https%3A//github.com/mullr/code-review-test/pull/1%23issuecomment-111586241%22%2C%20%22https%3A//github.com/mullr/code-review-test/commit/e2ba0277862212ae04b53994c5120f14cf57fc36%23commitcomment-11659881%22%2C%20%22https%3A//github.com/mullr/code-review-test/commit/2a57322e80eff058de68719e98fa99e94145ebf0%23commitcomment-11659923%22%2C%20%22https%3A//github.com/mullr/code-review-test/pull/1%23discussion_r32343588%22%2C%20%22https%3A//github.com/mullr/code-review-test/pull/1%23discussion_r32343811%22%2C%20%22https%3A//github.com/mullr/code-review-test/pull/1%23discussion_r32343954%22%2C%20%22https%3A//github.com/mullr/code-review-test/pull/1%23discussion_r32343998%22%2C%20%22https%3A//github.com/mullr/code-review-test/pull/1%23discussion_r32344208%22%2C%20%22https%3A//github.com/mullr/code-review-test/pull/1%23issuecomment-111590317%22%2C%20%22https%3A//github.com/mullr/code-review-test/pull/1%23issuecomment-111594144%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/mullr/code-review-test/pull/1%23issuecomment-111579380%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5B%20%5D%20this%20is%20a%20checkbox%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A17%3A29Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%2C%20%7B%22body%22%3A%20%22-%20%5B%20%5D%20this%20is%20really%20a%20checkbox%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A17%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%2C%20%7B%22body%22%3A%20%22This%20is%20a%20top-level%20comment%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A44%3A05Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%2C%20%7B%22body%22%3A%20%22A%20test%20thing%20%3Cb%3Ebold%3C/b%3E%22%2C%20%22created_at%22%3A%20%222015-06-12T19%3A06%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Cdetails%3E%5Cr%5Cn%20%20%3Csummary%3EPR%20v4%3C/summary%5Cr%5Cn%20%20%3Cdetails%3E%5Cr%5Cn%20%20%20%20%3Csummary%3E%28PDB-1234%29%20Change%20stuff%3C/summary%3E%5Cr%5Cn%20%20%20%20%3Cul%3E%5Cr%5Cn%20%20%20%20%20%20%3Cli%3EComment%20one%3C/li%3E%5Cr%5Cn%20%20%20%20%20%20%3Cli%3EComment%20two%3C/li%3E%5Cr%5Cn%20%20%20%20%3C/ul%3E%5Cr%5Cn%20%20%3C/details%3E%5Cr%5Cn%3C/details%3E%5Cr%5Cn%20%20%22%2C%20%22created_at%22%3A%20%222015-06-12T19%3A23%3A26Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%207fa5dd5fa7f8d77c99bafadb8f961ab7dc1aeb77%20test.txt%2016%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/mullr/code-review-test/pull/1%23discussion_r32343811%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22a%20files%20changed%20comment%20on%20the%20newest%20one%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A51%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%2C%20%7B%22body%22%3A%20%22%2B1%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A52%3A41Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10120698%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rbrw%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A53%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10120698%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rbrw%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A-1%3A%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A55%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10120698%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/rbrw%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20test.txt%3AL1-21%22%7D%2C%20%22Commit%20e2ba0277862212ae04b53994c5120f14cf57fc36%20test.txt%206%206%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/mullr/code-review-test/commit/e2ba0277862212ae04b53994c5120f14cf57fc36%23commitcomment-11659416%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22comment%20on%20a%20commit%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A16%3A31Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%2C%20%7B%22body%22%3A%20%22done%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A19%3A01Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A45%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20test.txt%3AL6%20%28e2ba027%29%22%7D%2C%20%22Pull%202a57322e80eff058de68719e98fa99e94145ebf0%20test.txt%207%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/mullr/code-review-test/pull/1%23discussion_r32343588%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20is%20a%20commit%20on%20the%20new%20%27files%20changed%27%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A48%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20test.txt%3AL1-19%22%7D%2C%20%22Commit%202a57322e80eff058de68719e98fa99e94145ebf0%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/mullr/code-review-test/commit/2a57322e80eff058de68719e98fa99e94145ebf0%23commitcomment-11659923%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22A%20comment%20on%20the%20new%20version%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A47%3A50Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%5D%2C%20%22title%22%3A%20%22Commit%202a57322%20comments%22%7D%2C%20%22Pull%209812d6b23956bb6ca9d55b773c338db7d3cc06ff%20test.txt%203%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/mullr/code-review-test/pull/1%23discussion_r32339849%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20is%20a%20comment%20in%20files%20changed%22%2C%20%22created_at%22%3A%20%222015-06-12T18%3A09%3A48Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/24409%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mullr%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20test.txt%3AL1-8%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 9812d6b23956bb6ca9d55b773c338db7d3cc06ff test.txt 3'></a> &nbsp;<b><a href='https://github.com/mullr/code-review-test/pull/1#discussion_r32339849'>File: test.txt:L1-8</a></b>
- <a href='https://github.com/mullr'></a> This is a comment in files changed
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/mullr/code-review-test/pull/1#issuecomment-111579380'>General Comment</a></b>
- <a href='https://github.com/mullr'></a> [ ] this is a checkbox
- <a href='https://github.com/mullr'></a> - [ ] this is really a checkbox
- <a href='https://github.com/mullr'></a> This is a top-level comment
- <a href='https://github.com/mullr'></a> A test thing <b>bold</b>
- <a href='https://github.com/mullr'></a> <details>
  <summary>
  
  PR v4</summary
  <details>
  <summary>
  
  (PDB-1234) Change stuff</summary>
  
  
  <ul>
  <li>Comment one</li>
  <li>Comment two</li>
  </ul>
  </details>
  </details>
- [ ] <a href='#crh-comment-Commit 2a57322e80eff058de68719e98fa99e94145ebf0'></a> &nbsp;<b><a href='https://github.com/mullr/code-review-test/commit/2a57322e80eff058de68719e98fa99e94145ebf0#commitcomment-11659923'>Commit 2a57322 comments</a></b>
- <a href='https://github.com/mullr'></a> A comment on the new version
- [ ] <a href='#crh-comment-Pull 2a57322e80eff058de68719e98fa99e94145ebf0 test.txt 7'></a> &nbsp;<b><a href='https://github.com/mullr/code-review-test/pull/1#discussion_r32343588'>File: test.txt:L1-19</a></b>
- <a href='https://github.com/mullr'></a> This is a commit on the new 'files changed'
- [ ] <a href='#crh-comment-Pull 7fa5dd5fa7f8d77c99bafadb8f961ab7dc1aeb77 test.txt 16'></a> &nbsp;<b><a href='https://github.com/mullr/code-review-test/pull/1#discussion_r32343811'>File: test.txt:L1-21</a></b>
- <a href='https://github.com/mullr'></a> a files changed comment on the newest one
- <a href='https://github.com/rbrw'></a> +1
- <a href='https://github.com/rbrw'></a> :+1:
- <a href='https://github.com/rbrw'></a> :-1:
- [x] <a href='#crh-comment-Commit e2ba0277862212ae04b53994c5120f14cf57fc36 test.txt 6 6'></a> &nbsp;<b><a href='https://github.com/mullr/code-review-test/commit/e2ba0277862212ae04b53994c5120f14cf57fc36#commitcomment-11659416'>File: test.txt:L6 (e2ba027)</a></b>
- <a href='https://github.com/mullr'></a> comment on a commit
- <a href='https://github.com/mullr'></a> done

<a href='https://www.codereviewhub.com/mullr/code-review-test/pull/1?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/mullr/code-review-test/pull/1?approve=1'></a>&nbsp;<a href='https://github.com/mullr/code-review-test/pull/1'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+asdfasd
+asd
+fa
```
Comment (mullr) [inline test.txt]: This is a comment in files changed

**Comment by mullr** [comment]
--- /dev/null
+++ b/test.txt
@@ -0,0 +1,13 @@
+x asdfasd
+x asd
+x fa
+x sdf
+x adsf
+x asdf
+x safsdfasdfasdf
+
+more
+lines
+at
+the
+bottom

**Comment by mullr** [comment]
```diff
--- /dev/null
+++ b/test.txt
@@ -0,0 +1,13 @@
+x asdfasd
+x asd
+x fa
+x sdf
+x adsf
+x asdf
+x safsdfasdfasdf
+
+more
+lines
+at
+the
+bottom
```

**Comment by mullr** [comment]
[ ] this is a checkbox

**Comment by mullr** [comment]
- [ ] this is really a checkbox

**Comment by mullr** [comment]
This is a top-level comment

File: test.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+lines 
+at
+the
+top
+
+x asdfasd
+x asd
```
Comment (mullr) [inline test.txt:7]: This is a commit on the new 'files changed'

File: test.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+x fa
+x sdf
+x adsf
+x asdf
+x safsdfasdfasdf
+
+more
+lines
+at
```
Comment (mullr) [inline test.txt:16]: a files changed comment on the newest one

Comment (rbrw) [inline test.txt]: +1

Comment (rbrw) [inline test.txt]: :+1:

Comment (rbrw) [inline test.txt]: :-1:

**Comment by mullr** [comment]
A test thing <b>bold</b>

**Comment by mullr** [comment]
<details>
  <summary>PR v4</summary
  <details>
    <summary>(PDB-1234) Change stuff</summary>
    <ul>
      <li>Comment one</li>
      <li>Comment two</li>
    </ul>
  </details>
</details>

**Pull request merged**